demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -203,10 +204,45 @@ def _search_for_pack_items(pack_id: str, items_list: list) -> list:
         """
         return list(filter(lambda s: next(iter(s.values())).get('pack') == pack_id, items_list))
 
+    @staticmethod
+    def _is_item_match(item_details: dict, base_condition: bool, exclude_ignored_dependencies: bool, marketplace: str) -> bool:
```

Comment:
Maybe a better naming for this function? took me a while to understand its purpose

Reply:
Care to suggest a name?

Reply:
`conditions_verifier`?
`should_add_item`?